modules/ROOT/pages/about-mule-message.adoc

```diff
@@ -185,6 +190,9 @@ In the previous variable declaration example, attribute `id` is declared without
 * `myVar."id"`
 * `myVar.`id``
 
+[[note_top_level_selectors]]
+Note that human-readable names, such as `Status Code`, that you see in console output for top-level attributes can differ from the name to use for selecting the top-level attribute, `statusCode`. Putting quotes around the human-readable name `Status Code` returns `null` instead of making that top-level attribute selectable. You can find the syntax to use for selecting the top-level attribute names, such as `Status Code`, `Reason Phrase`, and `Headers`, through xref:studio::datasense-concept.adoc#using-datasense-for-autocomplete[autocompletion in Studio] or in the connector documentation, for example, in xref:connectors::http/http-connector-xml-reference.adoc#http-request[HTTP Request] documentation.
```

Comment:
_... can differ from the name to use for selecting the top-level attribute, `statusCode`._  >

**... can differ from the actual name of the top-level attribute, `statusCode`.** (otherwise, it seems that we imply that names for selecting attributes are different, while the truth is that to select an attribute you must use the proper name, not what the UI or logs show) 

_Putting quotes around the human-readable name `Status Code` returns `null` instead of making that top-level attribute selectable._

I find that sentence misleading, because it makes me thing that the problem with *Status Code* is surrounding it with quotes, while the actual issue is that *Status Code* is a name for a non-existent value.